app/src/main/java/com/example/parkingsystemkotlin/mvp/model/reservation/ReservationInformationDB.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.parkingsystemkotlin.mvp.model.reservation
+
+import java.util.Calendar
+
+object ReservationInformationDB : DataBase {
+
+    private const val YEAR_START_AND_FINISH = 1970
+    private const val MONTH_START_AND_FINISH = 1
+    private const val DAY_OF_MONTH_START_AND_FINISH = 1
+    private const val HOUR_OF_DAY_START_AND_FINISH = 0
+    private const val MINUTE_START_AND_FINISH = 0
+    private const val SECURITY_CODE = ""
+    private const val PLACE = ""
+
+    private var reservationInfomation: MutableMap<String, MutableList<Reservation>> = mutableMapOf()
+
+    override fun getReservationsDB(place: String): MutableList<Reservation> = reservationInfomation[place] ?: mutableListOf()
+
+    override fun putReservationDB(startDateAndTime: Calendar, finishDateAndTime: Calendar, securityCode: String, place: String) {
+        var reservationsList: MutableList<Reservation>? = reservationInfomation[place]
```

Comment:
do you need this to be null? or can be not null?

Reply:
changed